src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -901,4 +902,144 @@ final case class EServicesApiServiceImpl(
       }
     }
   }
+
+  override def exportEServiceDescriptor(eserviceId: String, descriptorId: String)(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem],
+    toEntityMarshallerFile: ToEntityMarshaller[File]
+  ): Route = {
+
+    def extractConfig(eService: CatalogProcess.EService, descriptor: CatalogProcess.EServiceDescriptor): Json = {
+      val latestRiskAnalysis =
+        eService.riskAnalysis.sortBy(_.createdAt)(Ordering[OffsetDateTime].reverse).map(_.riskAnalysisForm).headOption
```

Comment:
> should we export only latest risk analysis? Couldn't they be different risk analysis for different purposes?

It would make sense, do you agree @beetlecrunch? Should we add all the risk analysis?

Reply:
Yes we must bring in all the risk analysis.